resilience4j-ratpack/src/main/java/io/github/resilience4j/ratpack/circuitbreaker/CircuitBreakerTransformer.java

```diff
@@ -77,7 +77,6 @@ public void error(Throwable throwable) {
 
                     @Override
                     public void complete() {
-                        circuitBreaker.releasePermission();
```

Comment:
This line should not be removed. It's only invoked when a call is cancelled

Reply:
ok, I will revert it then

Reply:
undo and re-commit changes